### PR TITLE
Added checkbutton for save plots option and text input for dependent …

### LINKED ISSUE #116: FI pipeline doesn't run due to missing config key
## Description
After sorting the page navigation and testing the pipeline pages, it turns out the FI pipeline won't run as the key for saving plots is never configured on the page. There are 2 solutions: 
1. add a widget to the page with the configuration enum as the key, or
2. the outputs are sort of the point of BioFEFI, maybe they should just always be saved?

## Tasks
If using 1.
- [ ] Add a widget to capture if the user wants to save FI and Fuzzy plots
- [ ] Put the "save output" options at the bottom of the page before the run button, like in "Train Models" page.

If using 2.
- [ ] Remove the widgets for saving outputs from all the pipeline pages
- [ ] Set any option relating to saving outputs to true in the `build_configuration` functions on the pipeline pages

In either case
- [ ] Test

### PR DESCRIPTION
**Pull request by EdAguilarB**
…variable name

## Linked Issues
- Closes #116

### CODE REVIEW

**Comment by dcl10** [comment]
@EdAguilarB Is this related to #116 ?

**Comment by EdAguilarB** [comment]
> @EdAguilarB Is this related to #116 ?

Yes!

File: biofefi/components/forms.py [hunk 1]
```diff
@@ -239,6 +239,20 @@ def fi_options_form():
             key=ConfigStateKeys.NumberOfTopRules,
         )
 
+    save_plots = st.checkbox(
```
Comment (dcl10) [inline biofefi/components/forms.py:242]: If you look at the ML page now that it's merged, do you think you could put all the options to save outputs at the bottom like on the ML page. Also could you use `st.toggle` for those options, like on the ML page pls?

**Review by dcl10 — CHANGES_REQUESTED** [review]

File: biofefi/services/ml_models.py [hunk 2]
```diff
@@ -32,8 +33,34 @@ def load_models(path: Path) -> dict[str, list]:
...
                 else:
-                    models[model_name] = [model_name]
+                    models[model_name] = [model]
         except UnpicklingError:
             pass  # ignore bad files
 
     return models
+
+
+def load_model_explain(path: Path, model_names: list) -> dict[str, list]:
```
Comment (dcl10) [inline biofefi/services/ml_models.py:43]:
```suggestion
def load_models_to_explain(path: Path, model_names: list) -> dict[str, list]:
```
⛏️

**Review by dcl10 — APPROVED** [review]

**Pull request merged**